config/locales/errors/en.yml

```diff
@@ -18,13 +18,13 @@ en:
       token_missing: token missing
     confirm_password_incorrect: Incorrect password.
     doc_auth:
-      acuant_network_error: Sorry we are experiencing technical difficulties.  Please
+      acuant_network_error: Sorry we are experiencing technical difficulties. Please
```

Comment:
double space was being dropped in the html making it hard to verify content present